tests/Jenkins/utilities.sh

```diff
@@ -332,6 +332,26 @@ function diracInstallCommand(){
 	./dirac-install -r `cat dirac.version` -t server -d
 }
 
+#.............................................................................
+#
+# prepareForServer:
+#
+#   This function gets the DIRAC install script
+#
+#.............................................................................
+
+function prepareForServer(){
+	echo '==> [prepareForServer]'
+
+	#Move to server directory
+	cd $SERVERINSTALLDIR
```

Comment:
i know it is just copy/paste, but woking with relative paths is always dangerous... cf "The Cleaner" :-)